network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt

```diff
@@ -85,4 +85,6 @@ enum class NetworkProtectionPixelNames(
     NETP_WAITLIST_NOTIFICATION_CANCELLED("m_netp_ev_waitlist_notification_cancelled_c"),
     NETP_WAITLIST_NOTIFICATION_LAUNCHED_DAILY("m_netp_ev_waitlist_notification_launched_d"),
     NETP_WAITLIST_NOTIFICATION_LAUNCHED("m_netp_ev_waitlist_notification_launched_c"),
+    NETP_WAITLIST_BETA_ENABLED_DAILY("m_netp_ev_waitlist_enabled_d"),
+    NETP_WAITLIST_BETA_ENABLED("m_netp_ev_waitlist_enabled_c"),
```

Comment:
This is the same as `m_netp_imp_settings_entry_[c/d]` from [here](https://app.asana.com/0/0/1205145153997698/f). But with different (better) naming for Android

Reply:
I assume we would want to add this in `NetpCohortPixelInterceptor`.

Reply:
Ah, great catch! Will do